packages/lesswrong/lib/vulcan-lib/utils.ts

```diff
@@ -401,7 +401,6 @@ export const sanitize = function(s: string): string {
       'calendly.com'
     ],
     allowedClasses: {
-      '*': footnoteClasses,
```

Comment:
I added this because at some point I was having trouble getting footnotes to work, because of the classes being stripped off tags that weren't `<span>`s. I didn't realise that this made things less permissive for tags that are not explicitly given here (which is why it broke the `<a>` tags of buttons). Anyway it appears the ultimate solution for footnotes didn't end up needing this, so I'm just removing it